testing/src/main/java/org/oppia/android/testing/KonfettiViewMatcher.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.android.testing
+
+import android.view.View
+import nl.dionsegijn.konfetti.KonfettiView
+import org.hamcrest.Description
+import org.hamcrest.TypeSafeMatcher
+
+class KonfettiViewMatcher {
+  companion object {
+    // Returns a matcher that matches for active confetti.
+    fun hasActiveConfetti(): TypeSafeMatcher<View> {
+      return ActiveConfettiMatcher()
+    }
+
+    // Custom class to check if a KonfettiView isActive().
```

Comment:
Can be a KDoc (though it doesn't need to be as detailed since this is a helper class).

Reply:
Should it be KDoc even though it's a private class? Alternatively, should this class not be private?

Reply:
I lean toward making this a KDoc--it's totally fine to document private classes & members (I actually do this all the time since it still serves a purpose for the target audience: fellow & future team members).

Reply:
Done